evalml/tests/conftest.py

```diff
@@ -100,13 +100,21 @@ def pytest_addoption(parser):
     parser.addoption("--has-minimal-dependencies", action="store_true", default=False,
                      help="If true, tests will assume only the dependencies in"
                      "core-requirements.txt have been installed.")
+    parser.addoption("--is-using-conda", action="store_true", default=False,
+                     help="If true, tests will assume that they are being run as part of"
+                          "the build_conda_pkg workflow with the feedstock.")
 
 
 @pytest.fixture
 def has_minimal_dependencies(pytestconfig):
     return pytestconfig.getoption("--has-minimal-dependencies")
 
 
+@pytest.fixture
+def is_using_conda(pytestconfig):
+    return pytestconfig.getoption("--is-using-conda")
```

Comment:
We have a separate conda feedstock PR to add this there, right?

I'm guessing we need that to be merged first in order for CI to work properly, yes?

Reply:
@dsherry Yes we'll have a separate PR for the feedstock that adds `--is-using-conda` to the command